hal/daler_dc_motor_ctrl.h

```diff
@@ -0,0 +1,58 @@
+/** 
+ * \page The MAV'RIC license
+ *
+ * The MAV'RIC Framework
+ *
+ * Copyright © 2011-2014
+ *
+ * Laboratory of Intelligent Systems, EPFL
+ */
+ 
+
+/**
+ * \file dc_motor_ctrl.h

```

Comment:
In addition, could it be renamed in something more generic or is it only for the board developed for the DALER?